README.md

```diff
@@ -14,19 +14,30 @@ Go get the repository
 
 Ensure your environment provides:
 
-- [go 1.12.9](https://golang.org/)
-- [golangci-lint 1.17.1](https://github.com/golangci/golangci-lint)
-- some working method of obtaining dependencies listed in `go.mod` (working internet access, `GOPROXY`)
-- env var `GOFLAGS="-mod=vendor"`
-- correctly configured cntlm or alpaca proxy running on localhost at port 3128 (only for updating vendor dependencies)
+- [go 1.13](https://golang.org/doc/install)
```

Comment:
maybe time to upgrade to 1.14.1 ?

Reply:
I think I'd like to keep the version of this at least consistent with Sysl, given that it's the same team of developers. Will create an issue for it though